### DISCUSSION TITLE
Increase icon sizes

### DISCUSSION THREAD

**Discussion by cantunborn** [Docile-Alligator/Infinity-For-Reddit#1004] [Ideas]
Subreddit icon sizes are quite small everywhere in the UI. In card layout, the icons are small even though there is a lot of empty space. Same for the subscription page, there is lots of empty space between 2 items in the list but still icons don't utilize it. 

It would be really nice if you could increase the icon sizes, since the icons would be easier/quicker to identify and there will be less wasted space.

**Comment by Docile-Alligator** [comment]
Good idea.
